usaspending_api/awards/serializers_v2/serializers.py

```diff
@@ -235,6 +235,7 @@ class Meta:
 
 
 class TransactionFPDSSerializerV2(LimitableSerializerV2):
+    small_business_competitive = serializers.CharField()
```

Comment:
What will this do?

Reply:
So converting `small_business_competitive` from Text to Boolean altered the API contract a bit.  Instead of returning "True"/"False", the API was returning True/False which was breaking a unit test.  This line of code causes the unit test to pass.  A couple alternatives would be to change the field back to Text and update the database or update the unit test to handle the change in the API return type.This one really got me wondering, though, how many other contracts were altered by these Django changes but were not caught by unit tests...